templates/01_common.conf.j2

```diff
@@ -11,17 +11,20 @@ NameVirtualHost *:80
      ProxyPass "http://cephas.centos.org/.well-known/acme-challenge/"
  </Location>
 
-# Default Header
-Alias /header /var/www/centos/header
```

Comment:
I'm all to keep the /header and /centos-design aliases for now, and remove *only* after we'll have updated one by one all the roles using this.